### PR TITLE
Sync person profile url from drupal.

### PR DESCRIPTION
**Pull request by awais786**
1. Add new field in people model.
2. Update data-loader and sync the profile url field.

ECOM-7587

screen shot 2017-04-04 at 4 46 25 pm

### CODE REVIEW

**Comment by awais786** [comment]
@tasawernawaz @waheedahmed @bderusha Please review.

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -205,6 +205,7 @@ class Person(TimeStampedModel):
         },
     )
     slug = AutoSlugField(populate_from=('given_name', 'family_name'), editable=True)
+    profile_url = models.URLField(null=True, blank=True)
```
Comment (waheedahmed) [inline course_discovery/apps/course_metadata/models.py:208]: Do we also need a default value for this?

**Review by waheedahmed — APPROVED** [review]
Just one question, otherwise 👍

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (awais786) [reply course_discovery/apps/course_metadata/models.py:208]: It has null=True so it will not effect the current records. 
@clintonb your thoughts ?

Comment (clintonb) [reply course_discovery/apps/course_metadata/models.py:208]: No default is needed.

**Review by clintonb — APPROVED** [review]

**Pull request merged**